Core/SiteRating.swift

```diff
@@ -158,7 +158,7 @@ public class SiteRating {
     }
     
     public func isFor(_ url: URL?) -> Bool {
-        return domain == url?.host
+        return self.url.host == url?.host
```

Comment:
This probably shouldn't have been using the domain var in the first place. This ensures both hosts are compared on equal terms.